src/stores/workspace/WorkspaceStore.ts

```diff
@@ -177,8 +178,51 @@ export default class WorkspaceStore {
 			const messages = await Promise.all(
 				messagesToLoad.map(id => this.api.messages.getMessage(id, this.attachedMessagesAC?.signal)),
 			);
+			const newStreams = messages
+				.map(message => message.sessionId)
+				.filter(
+					(stream, index, self) =>
+						index === self.findIndex(str => str === stream) &&
+						!this.messagesStore.filterStore.filter.streams.includes(stream),
+				);
+
+			messages
+				.map(message => message.sessionId)
+				.filter(
+					(stream, index, self) =>
+						index === self.findIndex(str => str === stream) && !newStreams.includes(stream),
```

Comment:
there is something wrong here.
there might be new attached messages with streams that already applied in filter. and it shows error that messages with session X are exluded, although those messages appear in the list